packages/common/src/hooks/useSortBy/useSortBy.ts

```diff
@@ -34,12 +35,13 @@ export const useSortBy = <T>({
     direction: getDirection(initialIsDesc),
   });
 
-  const onChangeSortBy = useCallback((newSortRule: SortRule<T>) => {
+  const onChangeSortBy = useCallback((newSortRule: Column<T>) => {
```

Comment:
here, and in the interface - `column: Column<T>` makes more sense to me than `newSortRule` since there is a different shape for a sortRule object. but 🤷

Reply:
Yeah you're right again!